sdks/java/extensions/zetasketch/src/test/java/org/apache/beam/sdk/extensions/zetasketch/HllCountTest.java

```diff
@@ -370,4 +464,14 @@ public void testExtractPerKey() {
         .containsInAnyOrder(Arrays.asList(KV.of("k", INTS1_ESTIMATE), KV.of("k", INTS2_ESTIMATE)));
     p.run();
   }
+
+  @Test
+  @Category(NeedsRunner.class)
+  public void testExtractPerKey_EmptySketch() {
```

Comment:
Since the Extract implementations for Global and PerKey are different, would test both types of empty sketches for both cases. Otherwise, very nice test coverage! :)

Reply:
Nice catch of the missing test! Done.